owncloudApp/src/main/java/com/owncloud/android/ui/activity/FileDisplayActivity.kt

```diff
@@ -1059,50 +1060,40 @@ class FileDisplayActivity : FileActivity(),
         replace: MutableList<Boolean?>,
         launchAction: (List<OCFile>, List<Boolean?>) -> Unit,
     ) {
-        val context = this@FileDisplayActivity
         if (!uiResult.data.isNullOrEmpty()) {
             var pos = 0
-            val fragmentManager = supportFragmentManager
+            var posDialog = data.lastIndex
             data.asReversed().forEachIndexed { index, file ->
                 val countDownValue = index + 1
 
-                val customDialog = FileAlreadyExistsDialog.Builder()
-                    .setTitle(
-                        this.getString(
-                            if (file.isFolder) {
-                                R.string.folder_already_exists
-                            } else {
-                                R.string.file_already_exists
-                            }
-                        )
-                    )
-                    .setDescription(
-                        this.getString(
-                            if (file.isFolder) {
-                                R.string.folder_already_exists_description
-                            } else {
-                                R.string.folder_already_exists_description
-                            }, file.fileName
-                        )
-                    )
-                    .setCheckboxText(this.getString(R.string.apply_to_all_conflicts, countDownValue.toString()))
-                    .build()
-
-
-                if (countDownValue > 1) {
-                    customDialog.checkboxVisible = true
-                }
+                val customDialog = FileAlreadyExistsDialog.newInstance(
```

Comment:
Add names for parameters here and the las coma.

Reply:
done